src/index.js

```diff
@@ -21,7 +21,13 @@ function base (ALPHABET) {
   var FACTOR = Math.log(BASE) / Math.log(256) // log(BASE) / log(256), rounded up
   var iFACTOR = Math.log(256) / Math.log(BASE) // log(256) / log(BASE), rounded up
   function encode (source) {
-    if (source instanceof Uint8Array) {} else if (ArrayBuffer.isView(source)) { source = new Uint8Array(source.buffer, source.byteOffset, source.byteLength) } else if (Array.isArray(source)) { source = Uint8Array.from(source) }
+    if (source instanceof Uint8Array) {
```

Comment:
Isn't this first `if` case already covered by the `if (!(source instanceof Uint8Array))` below?

Reply:
Oh wait, I think I got it. This way you don't need to do further checks.